.github/workflows/ci.yml

```diff
@@ -38,6 +38,16 @@ jobs:
         flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
         # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
         flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+    - name: Setup version bump env vars
```

Comment:
I think this is not necesary. There should be a `GITHUB_BASE_REF` environment variable. I would skip most of this. And just check `os.environ.get('GITHUB_BASE_REF')` in `run()`.

https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables